[
  {
    "message": "user with this data didn`t exist",
    "time": 1587755003377,
    "code": 404,
    "stackTrace": "UserNotFoundError: user with this data didn`t exist\n    at _callee2$ (/home/danyil/IdeaProjects/big_project_exam/server/src/controllers/queries/userQueries.js:18:11)\n    at tryCatch (/home/danyil/IdeaProjects/big_project_exam/server/node_modules/regenerator-runtime/runtime.js:45:40)\n    at Generator.invoke [as _invoke] (/home/danyil/IdeaProjects/big_project_exam/server/node_modules/regenerator-runtime/runtime.js:274:22)\n    at Generator.prototype.<computed> [as next] (/home/danyil/IdeaProjects/big_project_exam/server/node_modules/regenerator-runtime/runtime.js:97:21)\n    at asyncGeneratorStep (/home/danyil/IdeaProjects/big_project_exam/server/src/controllers/queries/userQueries.js:15:103)\n    at _next (/home/danyil/IdeaProjects/big_project_exam/server/src/controllers/queries/userQueries.js:17:194)"
  },
  {
    "message": "user with this data didn`t exist",
    "time": 1587755013831,
    "code": 404,
    "stackTrace": "UserNotFoundError: user with this data didn`t exist\n    at _callee2$ (/home/danyil/IdeaProjects/big_project_exam/server/src/controllers/queries/userQueries.js:18:11)\n    at tryCatch (/home/danyil/IdeaProjects/big_project_exam/server/node_modules/regenerator-runtime/runtime.js:45:40)\n    at Generator.invoke [as _invoke] (/home/danyil/IdeaProjects/big_project_exam/server/node_modules/regenerator-runtime/runtime.js:274:22)\n    at Generator.prototype.<computed> [as next] (/home/danyil/IdeaProjects/big_project_exam/server/node_modules/regenerator-runtime/runtime.js:97:21)\n    at asyncGeneratorStep (/home/danyil/IdeaProjects/big_project_exam/server/src/controllers/queries/userQueries.js:15:103)\n    at _next (/home/danyil/IdeaProjects/big_project_exam/server/src/controllers/queries/userQueries.js:17:194)"
  }
]